mitosheet/mitosheet/api/get_validate_snowflake_credentials.py

```diff
@@ -17,6 +17,16 @@
 except ImportError:
     SNOWFLAKE_CONNECTOR_IMPORTED = False
 
+# Global variable used to cache the snowflake credentials so that when users
+# open the snowflake taskpane multiple times in the same mito instantiation, 
```

Comment:
Update this to mention lifecycle of credentials. They exist for all mitosheets in this kernel until the kernel is restarted.